l10n_it_fatturapa_out/data/l10n_it_fatturapa_out_data.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="0">
+      <record forcecreate="True" id="decimal_unit_price_xmlpa" model="decimal.precision">
+            <field name="name">price_unit_xml</field>
```

Comment:
Userei per coerenza qualcosa tipo `Product Price for XML e-invoices`

Reply:
ok fatto